### ISSUE TITLE
Rubocop beautification doesn't work

### ISSUE THREAD

**Issue by vovanmix** [Glavin001/atom-beautify#1653]
# Description

Rubocop beautification doesn't work. I can see rubocop linters errors in the editor. If I run rubocop from the command line, it fixes all the errors.

# Input Before Beautification

This is what the code looked like before:

```
class Calendar < ActiveRecord::Base
  has_many :users, through: :events, source: :eventable, source_type: 'User'




  validates :name, presence: true
end

```

# Expected Output

The beautified code should have looked like this:

```
class Calendar < ActiveRecord::Base
  has_many :users, through: :events, source: :eventable, source_type: 'User'

  validates :name, presence: true
end
```

# Actual Output

The beautified code actually looked like this:

```
class Calendar < ActiveRecord::Base
  has_many :users, through: :events, source: :eventable, source_type: 'User'




  validates :name, presence: true
end
```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist: [Gist](https://gist.github.com/vovanmix/31ddb59fc659bd442c88991fd37774f8)

Tried to run this in terminal:
```
/Users/vladimir/.rbenv/shims/rubocop --auto-correct --config /Users/vladimir/projects/breakaleg/.rubocop.yml /var/folders/63/jzj7mjl17fg0hzkm8j8psf700000gq/T/temp11748-19165-1jm0d43.n8h7i4quxr
```
And it beautifies the tmp file correctly.

Tried to make the .rubocop.yml empty, tried on many different .rb files. Beautification just doesn't work.

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by eboyko** [comment]
Have the same issue

**Comment by Glavin001** [comment]
Duplicate of https://github.com/Glavin001/atom-beautify/issues/1652

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/1685 has been merged. Await next release.

**Comment by Glavin001** [comment]
Published to v0.30.0
